chainstate/src/detail/mod.rs

```diff
@@ -44,28 +52,25 @@ use logging::log;
 use utils::eventhandler::{EventHandler, EventsController};
 use utxo::UtxosDBMut;
 
-use self::{
-    orphan_blocks::{OrphanBlocksRef, OrphanBlocksRefMut},
-    time_getter::TimeGetter,
-};
+use self::orphan_blocks::{OrphanBlocksRef, OrphanBlocksRefMut};
 use crate::{detail::orphan_blocks::OrphanBlocksPool, ChainstateConfig, ChainstateEvent};
 
 type TxRw<'a, S> = <S as Transactional<'a>>::TransactionRw;
 type TxRo<'a, S> = <S as Transactional<'a>>::TransactionRo;
 type ChainstateEventHandler = EventHandler<ChainstateEvent>;
 
-const HEADER_LIMIT: BlockDistance = BlockDistance::new(2000);
+pub const HEADER_LIMIT: BlockDistance = BlockDistance::new(2000);
 
 pub type OrphanErrorHandler = dyn Fn(&BlockError) + Send + Sync;
 
 #[must_use]
 pub struct Chainstate<S> {
-    chain_config: Arc<ChainConfig>,
-    chainstate_config: ChainstateConfig,
-    chainstate_storage: S,
-    orphan_blocks: OrphanBlocksPool,
+    pub(crate) chain_config: Arc<ChainConfig>,
```

Comment:
Instead of these invasive pub(crate), why not create proper reference getters that are pub?

Reply:
Done.